gensim/models/base_any2vec.py

```diff
@@ -966,6 +989,9 @@ def train(self, sentences=None, input_streams=None, total_examples=None, total_w
             total_words=total_words, epochs=epochs, start_alpha=start_alpha, end_alpha=end_alpha, word_count=word_count,
             queue_factor=queue_factor, report_delay=report_delay, compute_loss=compute_loss, callbacks=callbacks)
 
+    def get_latest_training_loss(self):
+        return 0
```

Comment:
should raise `NotImplementet` (loss feature works only for w2v now, not for d2v/fasttext/etc) or  return `-1` maybe?

Reply:
I changed it to raise an exception. I think -1 will be confusing since it is not the value that will be displayed. The value that is displayed is the running training loss divided by the number of samples. That is why I chose to return 0 and -1. -1 will look like the loss is decreasing as the number of processed words increases.